tests/script-test.ts

```diff
@@ -29,9 +29,16 @@ import getRPCHandler, { HandlerConstructorConfig, RPCHandler } from "../dist/";
 
   const latencies = handler.getLatencies();
 
-  const provider = handler.getFastestRpcProvider();
-
-  console.log(provider);
+  // console.log("=====================================");
+  // console.log(networkIds);
+  // console.log("=====================================");
+  // console.log(networkNames);
+  // console.log("=====================================");
+  // console.log(networkRpcs);
+  // console.log("=====================================");
+  // console.log(networkCurrencies);
+  // console.log("=====================================");
+  // console.log(networkExplorers);
```

Comment:
Consider removing this unused code.

Reply:
This is a handy manual test script to have, I'd rather uncomment the logs and keep this because it's the best way to manually confirm everything is working